contrib/kind.sh

```
@@ -347,8 +316,11 @@ parse_args() {
                                                   ;;
             -adv | --advertise-default-network) ADVERTISE_DEFAULT_NETWORK=true
                                                   ;;
-            -ic | --enable-interconnect )       OVN_ENABLE_INTERCONNECT=true
-                                                ;;
+            -ce | --enable-central )              OVN_ENABLE_INTERCONNECT=false
+                                                  ;;
+            -ic | --enable-interconnect )         OVN_ENABLE_INTERCONNECT=true
+                                                  IC_ARG_PROVIDED=true
```

Shellcheck (0.10.0)

[warning] IC_ARG_PROVIDED appears unused. Verify use (or export if used externally).

(SC2034)